front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/adjustConfWithTrainToModify.ts

```diff
@@ -9,8 +9,8 @@ import { msToKmh } from 'utils/physics';
 
 export function adjustConfWithTrainToModifyV2(
   trainSchedule: TrainScheduleResult,
-  pathSteps: PathStep[],
-  rollingStockId: number,
+  pathSteps: (PathStep | null)[],
```

Comment:
You don't need to change the type here I think
```suggestion
  pathSteps: PathStep[],
```

Reply:
My bad, no you need to keep this sorry ! You can resolve this comment